Documentation/kubernetes/quickinstall.rst

```diff
@@ -66,15 +66,6 @@ chapter.
       [adjust the etcd address]
 
 
```

Comment:
Should we get rid of this file? These instructions seem error prone. Either we should remove it or add the etcd configuration guide here.

Reply:
Definitely out of scope of this PR

Reply:
I know, I was just asking if we should delete it or not.

Reply:
TBH, I think it's the best instructions we have right now. The other instructions are so verbose that nobody is following them.